KoreBuild-dotnet/build/_xunit-test.shade

```diff
@@ -51,6 +51,9 @@ projectFile=''
                         testArgs = " -parallel none";
                     }
 
+                    // Removing app domains to allow custom test discovery
+                    testArgs += " -noappdomain";
```

Comment:
This affects binding redirects (they would no longer work). What's the reason for this change?